* <grobian@gentoo.org>: upstream has a better/different fix, but it
                        involves a lot more patching as much more has
						changed since the release

--- pathfinder-0.20.0/compiler/include/qname.h
+++ pathfinder-0.20.0/compiler/include/qname.h
@@ -38,6 +38,14 @@
 
 /* PFns_t */
 #include "ns.h"
+/* boolean type `bool' and constants `true', `false' */
+#ifdef HAVE_STDBOOL_H
+#include <stdbool.h>
+#elif !defined(HAVE_BOOL)
+#define bool    char
+#define true    (char)1
+#define false   (char)0
+#endif
 
 typedef unsigned int PFqname_t ;
 
